okhttp/src/main/java/okhttp3/OkHttpClient.kt

```diff
@@ -303,6 +312,8 @@ open class OkHttpClient internal constructor(
     internal var readTimeout: Int = 10000
     internal var writeTimeout: Int = 10000
     internal var pingInterval: Int = 0
+    internal var webSocketCompression: Boolean = false
```

Comment:
this should be true by default?

Reply:
Setting it to true will silently enable compression for existing web socket users (if their servers supports it).
If you're sure it's appropriate, I can update it to be true by default.

Reply:
Lets do 2 PRs – first to check this all in, off by default. And a second PR that only flips the default.

OkHttp’s design objective includes this phrase “OkHttp is an HTTP client that’s efficient by default”. As far as I’m concerned that means compression is enabled by default.

Reply:
ALSO – can you think of any reason why end-users would want to disable compression? I think we should always advertise support for it and let the server decide.

Reply:
(ie. remove the fiddly configuration flags until we’re presented with a concrete use case justifying them.)

Reply:
I personally don't see any reasons why we should keep compression off. My only concern was the silent default change. 
Sure, let's go with your approach.